src/main/scala/chisel3/iotesters/FirrtlTerpBackend.scala

```diff
@@ -117,6 +117,8 @@ private[iotesters] object setupFirrtlTerpBackend {
       dutGen: () => T,
       optionsManager: TesterOptionsManager = new TesterOptionsManager): (T, Backend) = {
 
+    // Workaround to propagate Annotations generated from command-line options
+    firrtl.Driver.loadAnnotations(optionsManager)
```

Comment:
We need to make sure this gets into the release notes - if your code was relying on firrtl to call this internally, you now need to call it explicitly.

Reply:
I'm probably confused.  But isn't this going to give deprecation warnings.  Shouldn't the fix here be to use `getAnnotations` then explicitly add those the options manager.